src/main/java/com/example/example_mod/mixin/TitleScreenMixin.java

```diff
@@ -1,6 +1,6 @@
 package com.example.example_mod.mixin;
```

Comment:
Missed repackage, you may want to update this (or just remove this.)

Reply:
Nevermind, I see that was intentional :foxpple:

Reply:
[ Yeah I'm probably gonna go and remove the title screen mixin this afternoon idk why we kept it ]